Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -745,14 +808,17 @@ - (void)loadForTextDisplay
             [self.tapForMoreLabel autoPinEdge:ALEdgeTop toEdge:ALEdgeBottom ofView:self.textView],
             [self.tapForMoreLabel autoPinEdgeToSuperviewEdge:ALEdgeBottom withInset:self.textVMargin],
             [self.tapForMoreLabel autoSetDimension:ALDimensionHeight toSize:self.tapForMoreHeight],
-        ];
+        ]];
     } else {
-        self.contentConstraints = @[
+        [self.contentConstraints addObjectsFromArray:@[
+            [self.textBubbleImageView autoSetDimension:ALDimensionWidth toSize:textBubbleSize.width],
+            [self.textBubbleImageView autoSetDimension:ALDimensionHeight toSize:textBubbleSize.height],
+            [self.textBubbleImageView autoPinEdgeToSuperviewEdge:(self.isIncoming ? ALEdgeLeading : ALEdgeTrailing)],
```

Comment:
TBD but having looked at WA a bunch I think we should be mroe like them: both media + caption inside a single bubble, media content rounded but not quite touching the bubble edges.  All message bubbles "full width" except for short text-only messages. Media always "full width" and with clamped aspect ratio.

Reply:
I'm leaning towards that as well, but I didn't want to make a big design change until we'd discussed it, so this PR tries to largely matches the existing design.

The lions share of this work is compatible with that approach - a MessageCell can now contain both an attachment container and a text container.